packages/api-generator/src/maps/v-data-table.js

```diff
@@ -17,7 +17,16 @@ const TableHeader = {
 }
 
 const DataTableEvents = [
-  { name: 'click:row', source: 'v-data-table', value: 'any, { select: (value: boolean) => void, isSelected: boolean, expand: (value: boolean) => void, isExpanded: boolean }' },
+  {
+    name: 'click:row',
+    source: 'v-data-table',
```

Comment:
I know some of the @vuetifyjs/core-team enjoys very long lines but please format this and other instances 🙏